.coveragerc

```diff
@@ -3,13 +3,15 @@ source =
     arches/
 
 omit =
-    */python?.?/*
+    */__init__.py
+    */celery.py
+    */hosts.py
     */models/migrations/*
+    */python?.?/*
     */settings*.py
+    */updateproject.py
```

Comment:
unfortunately not, coverage fails without the line. What solution are you proposing? Inline-ignoring in the `updateproject` management command, or ignoring the folder as a whole?

Reply:
I was thinking we'd exclude the functions we're not developing anymore but continue measuring update_to_76, but now I realize that excluding other untested code is just going to raise the bar higher anyway, so never mind.

I'm fine with excluding this one mgmt command, but I just want to be able to say _why_ in case we start getting requests to exclude other ones. This one's a one-off utility that quickly becomes dead code. Something like that.

Reply:
I hear you, and I moved the coverage avoidance from `coveragerc` to using a `pragma` statement